integrations/pytorch/tutorials/sparse_transfer_learning_tutorial.md

```diff
@@ -0,0 +1,157 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Sparse Transfer Learning for Image Classification
+
+This tutorial shows how Neural Magic sparse models simplify the sparsification process by offering pre-sparsified models for transfer learning onto other datasets.
+
+## Overview
+
+Neural Magic’s ML team creates sparsified models that allow anyone to plug in their data and leverage pre-sparsified models from the SparseZoo. 
+Sparsifying involves removing redundant information from neural networks using algorithms such as pruning and quantization, among others. 
+This sparsification process results in many benefits for deployment environments, including faster inference and smaller file sizes. 
+
+Unfortunately, many have not realized the benefits due to the complicated process and number of hyperparameters involved.
+Working through this tutorial, you will experience how Neural Magic recipes simplify the sparsification process. In this tutorial you will:
+- Download and prepare a pre-sparsified image classification model.
+- Apply a sparse transfer learning recipe on the pre-sparsified model.
+
+The examples listed in this tutorial are all performed on the [Imagenette](https://github.com/fastai/imagenette) dataset.
+
+## Need Help?
+
+For Neural Magic Support, sign up or log in to get help with your questions in our Tutorials channel: [Discourse Forum](https://discuss.neuralmagic.com/) and/or [Slack](https://join.slack.com/t/discuss-neuralmagic/shared_invite/zt-q1a1cnvo-YBoICSIw3L1dmQpjBeDurQ). 
+
+## Setting Up
+
+This tutorial can be run by cloning and installing the `sparseml` repository which contains scripts and recipes for
+this example:
+
+```bash
+git clone https://github.com/neuralmagic/sparseml.git
+pip install sparseml[torchvision]
+```
+Note:  make sure to upgrade `pip` using `python -m pip install -U pip`
+## Downloading and Preparing a Pre-Sparsified Model
```

Comment:
We should be able to get by without having the user run the SparseZoo code to download. Specifically the vision.py script should accept the stubs to start from and auto download from the zoo, see the YOLOv5 tutorial as an example

Reply:
Resolved!